deploy/charts/harvester/values.yaml

```diff
@@ -79,6 +79,10 @@ minio:
   ##
   enabled: true
 
+  ## Specify the mode.
+  ##
+  mode: distributed
```

Comment:
will this mode value be applied or used?

Reply:
This maps to https://github.com/rancher/harvester/blob/0b460b13e2234cfcd3e160ad91053bf9c8135357/deploy/charts/harvester/charts/minio/values.yaml#L75